objects/client.c

```diff
@@ -3010,6 +3010,8 @@ client_unmanage(client_t *c, client_unmanage_t reason)
 
     if(reason != CLIENT_UNMANAGE_DESTROYED)
     {
+        xwindow_buttons_grab(c->window, &(button_array_t){ .len = 0 });
+        xwindow_grabkeys(c->window, &(key_array_t){ .len = 0 });
```

Comment:
Random drive by comment:
Both these functions do "ungrab everything, then re-grab all the requested stuff". Due to the empty argument, the re-grabbing doesn't do anything. Dunno if it would be simpler to just copy the "ungrab everything" request here.

But whatever. You still get my :+1: .